### PR TITLE
[Inventory] Remove Trader's Satchel ID from inventory.cpp

### PR DESCRIPTION
**Pull request by N0ctrnl**
Not sure if this is the right way to do this, but seems to work...

### CODE REVIEW

File: zone/inventory.cpp [hunk 1]
```diff
@@ -1673,7 +1673,7 @@ bool Client::SwapItem(MoveItem_Struct* move_in) {
 			if (dstbag)
 				dstbagid = dstbag->GetItem()->ID;
 		}
-		if (srcitemid==17899 || srcbagid==17899 || dstitemid==17899 || dstbagid==17899){
+		if ((srcbagid && srcbag->GetItem()->BagType == EQ::item::BagTypeTradersSatchel) || (dstbagid && dstbag->GetItem()->BagType == EQ::item::BagTypeTradersSatchel) || (srcitemid && src_inst->GetItem()->BagType == EQ::item::BagTypeTradersSatchel) || (dstitemid && dst_inst->GetItem()->BagType == EQ::item::BagTypeTradersSatchel)){
```
Comment (hgtw) [inline zone/inventory.cpp:1676]: Breaking each OR condition into a separate line would make this more readable

```cpp
if ((srcbagid && srcbag->GetItem()->BagType == EQ::item::BagTypeTradersSatchel) ||
    (dstbagid && dstbag->GetItem()->BagType == EQ::item::BagTypeTradersSatchel) ||
    (srcitemid && src_inst->GetItem()->BagType == EQ::item::BagTypeTradersSatchel) ||
    (dstitemid && dst_inst->GetItem()->BagType == EQ::item::BagTypeTradersSatchel)) {
```

Comment (N0ctrnl) [reply zone/inventory.cpp:1676]: Works for me

**Review by mackal — APPROVED** [review]

File: zone/inventory.cpp (hunk 1 above)
Comment (Akkadius) [reply zone/inventory.cpp:1676]: @N0ctrnl were you going to push the change for this prior to merge?

**Review by Akkadius — CHANGES_REQUESTED** [review]
Pending PR comment changes

File: zone/inventory.cpp (hunk 1 above)
Comment (N0ctrnl) [reply zone/inventory.cpp]: My bad. Done.

Comment (Akkadius) [reply zone/inventory.cpp]: @N0ctrnl no problem at all man! Thanks for pushing this <3

**Review by Akkadius — APPROVED** [review]

**Pull request merged**